### DISCUSSION TITLE
Dont show products images

### DISCUSSION THREAD

**Discussion by muhammetfaik** [solidusio/solidus#4994] [Troubleshooting]
I dont show products images in cloud domain.
I use fly.io. It's my fly.io link: https://tesetturgiyimson.fly.dev
I use digitalocean spaces.I make public one by one Digital ocean spaces files   after I upload products images.
But Product images appear at first, then they are deleted.
Its my products ima
![Screenshot from 2023-03-25 19-23-15](https://user-images.githubusercontent.com/15978174/227729497-80d929c7-7b3e-4d4e-ba06-b47d05f96458.png)
ge link screen:

Help Me!

**Comment by jarednorman** [comment]
You're likely generating signed, expiring URLs for the images which are then rendered in a cached portion of the page. The URLs expire, then the images stop working. Here is the ActiveStorage documentation for configuring public access: https://edgeguides.rubyonrails.org/active_storage_overview.html#public-access
